packages/picasso/src/RichText/hooks/useRichText/useRichText.tsx

```diff
@@ -64,18 +64,25 @@ const picassoMapper = (child: ReactNode): ReactNode => {
 
   const type =
     componentMap[child.type as keyof typeof componentMap] || child.type
-  const mappedChildren = child.props.children.map(picassoMapper)
+
+  const hasChildChildren = Boolean(child.props.children)
+
+  const mappedChildren = hasChildChildren
+    ? child.props.children.map(picassoMapper)
+    : null
 
   return createElement(type, { key: child.key }, mappedChildren)
 }
 
 const useRichText = (value: ASTType): ReactNode[] | ReactNode => {
   const mappedTextNodes = useMemo(() => {
-    if (!value.children) {
+    const { children } = value
+
+    if (!children || children.length === 0) {
```

Comment:
Could be simplfied to:
```suggestion
    if (!children?.length) {
```